code/parachain/frame/composable-traits/src/storage.rs

```diff
@@ -0,0 +1,58 @@
+use codec::{Decode, Encode};
+use scale_info::TypeInfo;
+
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+
+/// Semantic abstraction of `Option` for updating storage items in monolithic interfaces where not
+/// all fields need updated.
+#[derive(Decode, Encode, Debug, Clone, PartialEq, Eq, TypeInfo)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum UpdateValue<T> {
+	/// Value will **not** be set/updated - will remain the same.
+	DoNotSet,
+	/// Value will be set/updated - will be changed.
+	Set(T),
+}
+
+impl<T> UpdateValue<T> {
+	/// Returns the contained [`Set`] value or a default.
+	///
+	/// Consumes the `self` argument then, if [`Set`], returns the contained
+	/// value, otherwise if [`DoNotSet`], returns the [default value] for that
```

Comment:
where does `[default value]` link to?

Reply:
Links to here
https://doc.rust-lang.org/std/default/trait.Default.html#tymethod.default

Reply:
I don't believe that will work since there's no definition of `[default value]` anywhere in the comment as far as I can tell